api/types/system/info.go

```diff
@@ -73,6 +73,7 @@ type Info struct {
 	SecurityOptions     []string
 	ProductLicense      string               `json:",omitempty"`
 	DefaultAddressPools []NetworkAddressPool `json:",omitempty"`
+	FirewallBackend     [][2]string          `json:",omitempty"`
```

Comment:
We've not been great at doing this, and relied on Golang doing "whatever golang chose", but we should probably start setting explicit names for `json` to indicate "this is what we want the name to be when marshalled);

```suggestion
	FirewallBackend     [][2]string          `json:"FirewallBackend,omitempty"`
```

Reply:
Dropped for now, but will-do when I get back to it.